### PR TITLE
#9: fixed port exception

### PR DESCRIPTION
**Pull request by ggajos**
for #9

### CODE REVIEW

**Comment by pinaf** [comment]
@ggajos if the first argument is not an integer the program just fails with an exception, correct?

**Comment by karato** [comment]
Many thanks, I will find someone to review it before we merge

**Comment by karato** [comment]
@longtimeago review it

File: src/main/java/com/nerodesk/Launch.java [hunk 1]
```diff
@@ -73,6 +73,20 @@ public static void main(final String... args) throws Exception {
     }
 
     /**
+     * Return passed port or run on default one 8080.
+     * @param args Command line args
+     * @return Application port.
+     */
+    private static int port(final String... args) {
```
Comment (longtimeago) [inline src/main/java/com/nerodesk/Launch.java]: do we really need to extract this code in a separate method?
I could say YES if:
- we are going to reuse it
- main method is too big (say, over 30 lines)

In this case, I would rather inline it

Comment (ggajos) [inline src/main/java/com/nerodesk/Launch.java]: Well, it's the first time in my life that somebody complain that method it too short. For me method is too big when it's more than 7 lines. Main methods have tendency to grow so I'm trying to extract that soon enough.

**Comment by ggajos** [comment]
@pinaf We don't have framework yet so I didn't want to overdue this. Application is starting on default port when not provided and complaining (exception) when parameters are incorrect. You might want to create another bug for this but let's for framework, no?

File: src/main/java/com/nerodesk/Launch.java (hunk 1 above)
Comment (longtimeago) [inline src/main/java/com/nerodesk/Launch.java]: @ggajos This is a "holy war" question :)
IMHO, 7 lines is too small. Extracting code from such a small method to the new method creates classic "spaghetti code" and disperse the logic. We need a balance everywhere.

Also, I know Yegor's preferences :)

**Comment by ggajos** [comment]
@longtimeago ok :), please see now

**Comment by longtimeago** [comment]
@rultor please, merge

**Comment by ggajos** [comment]
@rultor merge

**Comment by ggajos** [comment]
@rultor merge

**Comment by karato** [comment]
@longtimeago thanks, paid, **16 mins** to your account, payment ID is `52442166`; review comments (c=1) added as a bonus; added +16 to your rating, now it is equal to [+2122](http://www.netbout.com/b/35024?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**